src/Domain/FormGroups/FormGroupGateway.php

```diff
@@ -143,4 +143,26 @@ public function getFormGroupByID($gibbonFormGroupID)
 
         return $this->db()->selectOne($sql, $data);
     }
+
+    /**
+     * Take a form group, and return the next one, or false if none.
+     *
+     * @version v17
+     * @since   v17
+     *
+     * @param int $gibbonFormGroupID
+     *
+     * @return int|false
+     */
+    public function getNextFormGroupID($gibbonFormGroupID)
+    {
+        $sql = 'SELECT gibbonFormGroupIDNext FROM gibbonFormGroup WHERE gibbonFormGroupID=:gibbonFormGroupID';
+        $result = $this->db()->select($sql, [
+            'gibbonFormGroupID' => $gibbonFormGroupID,
+        ]);
+        if ($result->rowCount() == 1) {
+            return $result->fetchColumn() ?? false;
+        }
+        return false;
```

Comment:
```suggestion
        return $this->db()->selectOne($sql, [
            'gibbonFormGroupID' => $gibbonFormGroupID,
        ]);
```

Reply:
The method here returns either the (int) ID or FALSE if not found. It's not exactly match what selectOne returns (an array of all columns or an empty array).